react/game/src/app/index.js

```diff
@@ -1,9 +1,10 @@
-import React, { Component, Fragment } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import Login from './screens/Login';
 import Game from './screens/Game';
+import PrivateRoute from './components/PrivateRoute';
 
 class App extends Component {
   state = { loggedin: false };
```

Comment:
U must store this value in a store of redux. Remember, think big. Maybe in the future u need to read the `loggedIn` value from any place and any component